samples/ServiceBusSedingToQueue/CustomLogginService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus;
+using ServiceBus.LogginPlugin;
+using ServiceBus.LogginPlugin.Abstractions;
+
+namespace ServiceBusSedingToQueue
+{
+    public class CustomLogginService : ILogginService
+    {
+        public void Dispose()
+        {
+            throw new NotImplementedException();
```

Comment:
A different type of problem, but it's confusing to see `IDisposable` implemented just to  throw an exception. Begs a question what for the `IDisposable` is really needed.

Reply:
@SeanFeldman this is just the test custom loggin service, no a real implementation

Reply:
I'm not a big fan of mixing "production" code with "testing" code. If possible, I always try to make testing code internal so that it doesn't leak out.

Reply:
Yes it was just a sample of how to use it, but I'll keep the storage sample for the repository.